debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -616,50 +619,75 @@ public static GtidNewChannelPosition parse(String value, String defaultValue) {
             .withImportance(Importance.LOW)
             .withValidation(Field::isClassName)
             .withDescription("JDBC Driver class name used to connect to the MySQL database server.");
+
     /**
      * A comma-separated list of regular expressions that match database names to be monitored.
-     * May not be used with {@link #DATABASE_BLACKLIST}.
+     * Must not be used with {@link #DATABASE_BLACKLIST}.
      */
-    public static final Field DATABASE_WHITELIST = Field.create(DATABASE_WHITELIST_NAME)
+    public static final Field DATABASE_INCLUDE_LIST = Field.create(DATABASE_INCLUDE_LIST_NAME)
             .withDisplayName("Databases")
             .withType(Type.LIST)
             .withWidth(Width.LONG)
             .withImportance(Importance.HIGH)
-            .withDependents(TABLE_WHITELIST_NAME)
+            .withDependents(TABLE_INCLUDE_LIST_NAME, TABLE_WHITELIST_NAME)
             .withDescription("The databases for which changes are to be captured");
 
+    /**
+     * Old, backwards-compatible "whitelist" property.
+     */
+    public static final Field DATABASE_WHITELIST = Field.create(DATABASE_WHITELIST_NAME)
+            .withDisplayName("Databases")
+            .withType(Type.LIST)
+            .withWidth(Width.LONG)
+            .withImportance(Importance.HIGH)
```

Comment:
@rk3rn3r Shouldn't be the importance of oboslete fields reduced to `LOW`? It might have an impact on UI side.

Reply:
Yep, I missed that change. Will do!
Adding the InvisibleRecommender is a good idea! then the prefix isn't really needed, correct?

Reply:
Done and applied the prefix and InvisibleRecommender everywhere